backend/src/contaxy/managers/auth.py

```diff
@@ -885,6 +885,7 @@ def create_user(
             id=user_id,
             technical_user=technical_user,
             created_at=datetime.now(timezone.utc),
+            has_password=True if bool(user_input.password) else False,
```

Comment:
Instead of `True if bool(user_input.password) else False` you could just write `bool(user_input.password)` or maybe more clearly `user_input.password is not None`

Reply:
I think the password field is already deleted at this point (see line 872)